src/models/Column.js

```diff
@@ -45,6 +45,10 @@ const Column = new mongoose.Schema( {
         type: String,
         required: true,
     },
+    isDoneColumn: {
```

Comment:
make use of the ```columnType``` property and set that to the value "DONE" or for backlog/todo column use "TODO"